### PR TITLE
printing in a file added

### PR DESCRIPTION
**Pull request by loco-eng**
Hi, can you review my code?

### CODE REVIEW

File: HelloWorldProject/Source.cpp [hunk 1]
```diff
@@ -10,11 +11,17 @@ class HelloPrinter {
...
+// Prints "Hello World!" in file1.txt
 int main() {
 
 	HelloPrinter p;
+	std::ofstream ofs("file1.txt");
+	
+	if (!ofs)
+		return -1;
+
+	p.print(ofs);
```
Comment (Scvirel) [inline HelloWorldProject/Source.cpp:24]: Return this what was deleted

Comment (loco-eng) [reply HelloWorldProject/Source.cpp:24]: Is it correctly?

File: HelloWorldProject/Source.cpp [hunk 2]
```diff
@@ -10,11 +11,17 @@ class HelloPrinter {
```
Comment (Scvirel) [inline HelloWorldProject/Source.cpp:12]: Pidor

Comment (Scvirel) [reply HelloWorldProject/Source.cpp:12]: Asugdayu

Comment (Scvirel) [reply HelloWorldProject/Source.cpp:12]: qwe

File: HelloWorldProject/Source.cpp (hunk 1 above)
Comment (Scvirel) [reply HelloWorldProject/Source.cpp:24]: qwe

**Review by Scvirel — COMMENTED** [review]
Fix

**Pull request merged**